theme.json

```diff
@@ -210,10 +210,22 @@
 					}
 				}
 			},
+			"core/post-excerpt": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)"
+				}
+			},
 			"core/post-date": {
 				"typography": {
 					"fontSize": "var(--wp--preset--font-size--small)",
-					"fontStyle": "italic"
+					"fontWeight": "400"
+				},
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "none"
+						}
+					}
```

Comment:
Removing underlines by default is not awesome but I think it is the least bad option.